stacks/loki/README.md

```diff
@@ -24,7 +24,7 @@ A Loki-based logging stack consists of 3 components:
 
 | Package               | Application Version   | Helm Chart Version |License                                                                                    |
 | ---| ---- | ---- | ------------- |
-| Loki Stack | 2.1.0 | [2.5.1](https://artifacthub.io/packages/helm/grafana/loki-stack/2.5.1) | [Apache 2.0](https://github.com/grafana/loki/blob/main/LICENSE) |
+| Loki Stack | 2.6.3 | [2.1.0](https://artifacthub.io/packages/helm/grafana/loki-stack/2.6.3) | [Apache 2.0](https://github.com/grafana/loki/blob/main/LICENSE) |
```

Comment:
Here, you swapped application version with Helm chart version. The application version is **2.1.0** and Helm chart version is **2.6.3**. Just check the [artifacthub.io page for Loki](https://artifacthub.io/packages/helm/grafana/loki-stack/2.6.3).

Reply:
Latest commit has everything